src/io-msbc.c

```diff
@@ -0,0 +1,253 @@
+/*
+ * BlueALSA - io-msbc.c
+ * Copyright (c) 2017 Arkadiusz Bokowy
+ *
+ * This file is a part of bluez-alsa.
+ *
+ * This project is licensed under the terms of the MIT license.
+ *
+ */
+
+#include "io.h"
+
+#include <ctype.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <math.h>
+#include <poll.h>
+#include <signal.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+#include <unistd.h>
+#include <sys/eventfd.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+
+#include <sbc/sbc.h>
+#include "hfp.h"
+#include "bluealsa.h"
+#include "transport.h"
+#include "utils.h"
+#include "io-msbc.h"
+#include "shared/log.h"
+
+#if defined(ENABLE_MSBC)
```

Comment:
For feature preprocessor flags please use single "if" (e.g. `#if ENABLE_MSBC`). Using "if define" might lead to unintentional code inclusion if one sets `#define ENABLE_MSBC 0`. I know, that someone might use `-DENABLE_MSBC` as an argument for gcc, but since we are using autotools for configuration, it's not the case here.